src/user/userData.ts

```diff
@@ -1,8 +1,10 @@
 import { request, gql, GraphQLClient } from 'graphql-request'
-import { userQuery } from '../constants/graphqlQueries';
+import { generateL2RasterProductQuery, userProductsQuery, userQuery } from '../constants/graphqlQueries';
+import { CurrentUser, Product, UserResponse } from '../types/graphqlTypes';
+import { GeneratedProduct, getUserProductsResponse } from '../types/constantTypes';
 
 const userIdQuery = gql`${userQuery}`
```

Comment:
Is there a specific reason why userQuery needs to be redeclared here?

Reply:
I think that was left behind and not necessary